Build CentOS7.8 images

brtkwr opened the pull request.

brtkwr committed:
Build CentOS7.8 images

priteau reviewed (approved):
LGTM. If I was pedantic I would say the system advertises itself as `7.8.2003`

brtkwr committed:
Remove lustre-client-ohpc from OpenHPC image

Without removing this, DIB fails because lustre is not supported by this
distro [0].

[0] https://wiki.whamcloud.com/display/PUB/Lustre+Support+Matrix
